app/models/comment.rb

```diff
@@ -56,11 +56,11 @@ def text= text
   end
 
   def people_allowed_to_be_mentioned
-    @people_allowed_to_be_mentioned ||= if parent.public?
-                                          "all"
-                                        else
-                                          [*parent.participants, parent.author].map(&:id).uniq
-                                        end
+    if parent.public?
+      "all"
+    else
+      [*parent.participants, parent.author].map(&:id).uniq
```

Comment:
Hmm, participations and poll_participations are generally non-visible. Comments and likes are visible. So there are few sense to include the former: strictly saying participation means "following" the discussion rather than participating in it and nobody sees your participation. On contrary comments are visible participation in the discussion. Likes are similar to comments in this respect. E.g. a user notices someone has liked the post and wants to say `@dude why do you like it?`

So, I guess, if changing that, I'd prefer 
`[*parent.comments.pluck(:author_id), *parent.likes.pluck(:author_id), parent.author_id].uniq`

Right?